gravitee-apim-e2e/api-test/policies/mock-policy.spec.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { afterAll, beforeAll, describe, expect } from '@jest/globals';
+import { APIsApi } from '@management-apis/APIsApi';
+import { forManagementAsApiUser } from '@client-conf/*';
+import { ApisFaker } from '@management-fakers/ApisFaker';
+import { ApiEntity } from '@management-models/ApiEntity';
+import { UpdateApiEntityFromJSON } from '@management-models/UpdateApiEntity';
+import { PlansFaker } from '@management-fakers/PlansFaker';
+import { PlanEntity } from '@management-models/PlanEntity';
+import { RuleMethodsEnum } from '@management-models/Rule';
+import { noContent, succeed } from '../../lib/jest-utils';
+import { LifecycleAction } from '@management-models/LifecycleAction';
+import { fetchGateway } from '../../lib/gateway';
+import { PathOperatorOperatorEnum } from '@management-models/PathOperator';
+
+const apiManagementApiAsApiUser = new APIsApi(forManagementAsApiUser());
+const orgId = 'DEFAULT';
+const envId = 'DEFAULT';
+let createdApi: ApiEntity;
+let createdPlan: PlanEntity;
+
+describe('Mock policy', () => {
+  describe('On api v1 - based on paths', () => {
+    beforeAll(async () => {
+      const newPlanEntity = PlansFaker.newPlan();
```

Comment:
To make it simpler, maybe we could create the plan as published, using :
`PlansFaker.newPlan({ status: PlanStatus.PUBLISHED })`

And so, remove the "publishApiPlan" call, in both test cases

Reply:
✅